.github/workflows/upload-dev-image.yml

```diff
@@ -40,24 +40,20 @@ jobs:
           go-version: ${{ matrix.go-version }}
           cache-dependency-path: vmware-event-router/go.sum
         id: go
-        
-      - id: login_to_gcp
-        uses: google-github-actions/auth@35b0e87d162680511bf346c299f71c9c5c379033
+      - name: Log in to the Container registry
+        uses: docker/login-action@v1
         with:
-          credentials_json: '${{ secrets.GCP_CREDENTIALS }}'
-
-      - name: Set up gcloud
-        uses: google-github-actions/setup-gcloud@e30db14379863a8c79331b04a9969f4c1e225e0b
-
-      - name: Configure Docker
-        run: gcloud auth configure-docker --quiet
-
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+        env:
+          VES_DOCKER: /usr/bin/docker
       - name: Get short COMMIT and TAG
         run: |
           echo "KO_COMMIT=$(echo -n $GITHUB_SHA | cut -c -8)" >> $GITHUB_ENV
           echo "KO_TAG=$(basename "${{ github.ref }}")" >> $GITHUB_ENV
-
       - name: Build and push multi-arch image
+        id: build
         run: |
           # build and push with development tag
-          ko resolve --platform=linux/arm64,linux/amd64 --tags ${KO_COMMIT},development -BRf deploy/event-router-k8s.yaml
\ No newline at end of file
+          ko resolve --platform=linux/arm64,linux/amd64 --tags ${KO_COMMIT},${KO_TAG} -BRf deploy/event-router-k8s.yaml
```

Comment:
why the change from `development` to `tag`?

Reply:
the TAG will be `development` but I will revert this change.